src/makeCalculator.js

```diff
@@ -1,10 +1,47 @@
 'use strict';
 
-/**
- * @return {object}
- */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add(number) {
+      this.result += number;
+
+      return this;
+    },
+
+    subtract(number) {
+      this.result -= number;
+
+      return this;
+    },
+
+    multiply(number) {
+      this.result = this.result * number;
+
+      return this;
+    },
+
+    divide(number) {
+      this.result = this.result / number;
```

Comment:
Consider using the shorthand operator for division assignment: `this.result /= number;` as suggested in the checklist.